src/screens/FormField.js

```diff
@@ -546,11 +561,28 @@ const FormFieldPage = () => (
           <GenericColor />
         </Property>
 
+        <Property name="formField.focus.containerFocus">
+          <Description>
+            Whether the focus should be on the FormField container or the Input.
+            When true, the FormField container will receive focus; when false,
+            the Input field will receive focus.
```

Comment:
@jcfilben let me know if you think this is better.

Reply:
Yeah I think that is good